src/main/java/randoop/operation/TypedClassOperation.java

```diff
@@ -116,7 +117,7 @@ public TypedClassOperation applyCaptureConversion() {
   }
 
   /**
-   * Produces a Java source code representation of this statement and append it to the given
```

Comment:
In `apply()` on line 101/102 above, please improve the Javadoc.  It wasn't clear to me whether this was related somehow to the type variables in the declaringClass, if the declaringClass is generic.  The documentation for the overridden method is clearer, but then I'm not sure exactly how this overriding implementation is different.